python/ray/experimental/collective/allreduce.py

```diff
@@ -79,11 +78,25 @@ def __call__(
         self,
         tensor,
         group_name: str = "default",
-        op: ray_collective_types.ReduceOp = ray_collective_types.ReduceOp.SUM,
+        op: ReduceOp = ReduceOp.SUM,
```

Comment:
For this, I was hoping we can use the existing `ray.util.collective` type as the user-facing one, not the new one.

Reply:
Ack. Done.